linera-base/src/crypto.rs

```diff
@@ -430,6 +430,12 @@ impl CryptoHash {
     pub fn as_bytes(&self) -> &HasherOutput {
         &self.0
     }
+
+    /// Returns the hash of `TestSignable(s)`, for testing purposes.
+    #[cfg(any(test, feature = "test"))]
```

Comment:
Optional: same as suggested on PR #1694, we can migrate to a new `with_testing` alias as used in other crates.

Another option is to do that everywhere in a separate PR :+1: